docs-ref-conceptual/azure-cli-extensions-overview.md

```diff
@@ -43,6 +45,22 @@ az extension add --source <URL-or-path>
 Once an extension is installed, it's found under the value of the `$AZURE_EXTENSION_DIR` shell variable. If this variable is unset, by default the value is `$HOME/.azure/cliextensions` on
 Linux and macOS, and `%USERPROFILE%\.azure\cliextensions` on Windows.
 
+### Install extensions automatically
+
+Azure CLI also has the ability to install an extension dynamically based on the command you run if the command is from an extension that is not installed yet. You can enable the feature through configuration:
+```azurecli-interactive
+az config set extension.use_dynamic_install=yes_prompt
+```
+To use for automation without prompt:
+```azurecli-interactive
+az config set extension.use_dynamic_install=yes_without_prompt
+```
+
+By default, the command stops after the needed extension is installed and you need to rerun the command. You can make the command continue to run after the extension is installed:
```

Comment:
By default, an extension command that prompts dynamic install will not continue to run.  You can change the default behavior and make the command continue by setting the **run_after_dynamic_install** property to **yes**.